src/Logger-PREVAC_TMC13/main.py

```diff
@@ -170,11 +173,15 @@ def connect(self) -> None:
         )
 
         self.register_host()
-        self.assign_master()
+        self.unique_identifier = "PREVAC_TMC13 - " + self.port_string
+        if self.unique_identifier not in self.device_communication:
```

Comment:
updated the driver, it was still working because this way release_master was never called which was not a problem. Could be only a problem if another instrument tries to access the device.